tests/integration/ef_tests/utils.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os
+
+from tests.utils.uint256 import uint256_to_int
+
+# Root of the GeneralStateTest in BlockchainTest format
+EF_GENERAL_STATE_TEST_ROOT_PATH = (
+    "./tests/integration/ef_tests/test_data/BlockchainTests/GeneralStateTests/"
+)
+
+DEFAULT_NETWORK = "Shanghai"
+
+
+def display_storage(uint256_tuple):
+    return hex(uint256_to_int(*uint256_tuple))
+
+
+def is_account_eoa(state: dict) -> bool:
+    return state.get("code") in [None, "0x"] and not state.get("storage")
+
+
+def process_json_file(filepath):
```

Comment:
is this processing a json file? a test file maybe? there is no indication that the main thing here is filtering out the tests. `get_test_xx` ?